### PR TITLE
f-header@9.2.0 - remove ml theme style overrides

### PR DESCRIPTION
**Pull request by dandel10n**
### Removed
- 'ml' theme override for Logo.
- width css definition from logo icon to make sure that icons for different tenants (je and ml) are rendered correctly with same height.
- nb-No and da-DK from copy, storybook story and tests.

### Changed
- Fixed Logo tests.

### Updated
- fozzie package to the latest (v6.2.0).

### CODE REVIEW

File: packages/components/organisms/f-header/src/components/_tests/Logo.test.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Logo from '../Logo.vue';
 describe('Logo', () => {
     it('should be defined', () => {
         const propsData = {
-            theme: 'je',
+            theme: 'jet',
```
Comment (DamianMullins) [inline packages/components/organisms/f-header/src/components/_tests/Logo.test.js]: Do we still need this prop?

Comment (dandel10n) [reply packages/components/organisms/f-header/src/components/_tests/Logo.test.js]: We need this prop for setting the right logo, but actually I noticed that some tests should fail with changing `je` to `jet` but they don't :) I will dig into this :+1:

Comment (DamianMullins) [reply packages/components/organisms/f-header/src/components/_tests/Logo.test.js]: Ah right, cool. OK, I'll keep an eye on any changes to the PR.

Comment (dandel10n) [reply packages/components/organisms/f-header/src/components/_tests/Logo.test.js]: Ok, I think now the changes make more sense :)

File: packages/components/organisms/f-header/src/components/_tests/Logo.test.js [hunk 2]
```diff
@@ -17,51 +16,48 @@ describe('Logo', () => {
         // Arrange
         const propsData = {
             theme: 'ml',
-            headerBackgroundTheme: 'transparent',
```
Comment (AlixRichards) [inline packages/components/organisms/f-header/src/components/_tests/Logo.test.js:20]: Do these both still need the `theme` to be set?

**Review by AlixRichards — APPROVED** [review]

**Review by DamianMullins — APPROVED** [review]

**Pull request merged**